test/reporters/list.spec.js

```diff
@@ -138,15 +139,15 @@ describe('List reporter', function() {
       var actual = {a: 'actual'};
       var expected = {a: 'expected'};
       var checked = false;
-      var stringifyDiffs = 'stringifyDiffObjs';
+      var callbackWhichStringifies = 'stringifyDiffObjs';
       var err;
       test = {};
       runner = createMockRunner('fail', 'fail', null, null, test);
       runner.on = function(event, callback) {
         if (
-          callback.toString().includes(stringifyDiffs) && // target correct fail event callback
           !checked &&
-          event === 'fail'
+          event === 'fail' &&
+          callback.toString().includes(callbackWhichStringifies) // target correct fail event callback
```

Comment:
How about just ditch the variable altogether?
```js
          callback.toString().includes('stringifyDiffObjs') // target correct fail event callback
```

Reply:
Sure will do.